bigtable/google/cloud/bigtable/table.py

```diff
@@ -184,6 +184,16 @@ def create(self):
                                                 table_id=self.table_id,
                                                 table={})
 
+    def exist(self):
```

Comment:
Our convention would be to spell this method name `exists`. The docstring should be in the form:

```python
    def exists(self):
        """Check whether the table exists.

        :rtype: bool
        :returns: True if the table exists, else False.
        """"
```